svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/service/FieldExtractor.java

```diff
@@ -315,6 +347,23 @@ public static AbdBloodPressure extractBloodPressure(Observation observation) {
           });
     }
 
+    // Set default systolic and diastolic blood pressure value if not exist to keep consistence with
+    // MAS BP reading.
+    if (result.getDiastolic() == null) {
```

Comment:
I am confused. The ticket specifically says if no blood pressure to ignore the resource. Yet you are adding a blood pressure with no meaningful value that will. print as "-/-". Why? Please do not add anything if both measurements are missing.

Reply:
Per our discussion, I change the code to skip complete empty BP reading, and default to 0 for missing one if one value exists.